templates/minimal/README.md

```diff
@@ -76,6 +83,53 @@ Development chains:
 * 💰 Are pre-configured with a genesis state that includes several pre-funded development accounts.
 * 🧑‍⚖️ One development account (`ALICE`) is used as `sudo` accounts.
 
+**Note**: running multiple nodes with the same command used for the single node setup is also possible and
+it can work up to a certain moment. The nodes will be peers, taking their turn in block production if manual
+seal is configured to allow nodes to produce blocks at certain intervals and in the meantime to
+import the blocks produced by peers. However, there is a big chance that at some point in time at least two nodes
+will overlap with the block production at a certain height, at which point they will fork and will not consider
+each others blocks anymore (stopping from being peers). They will continue to participate in blocks production
+of their own fork and possibly of other nodes too.
+
+### Omni Node based development chain
+
+Minimal template can run as a single-node chain based on Omni Node (TODO: add link to doc) too.
+
+* ⬇️  Omni Node (TODO: add link to omni node docs) can be run by using the `polkadot-parachain` binary, which
+can be downloaded from [Polkadot SDK releases](https://github.com/paritytech/polkadot-sdk/releases/latest).
+It is one instantiation of the idea of OmniNode, which comes with [builtin customized chain specs](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus/polkadot-parachain/chain-specs).
+
+* 🔗 Once built, add it to the `PATH` environment variable like so:
+
+```sh
+export PATH="<path-to-binary>:$PATH"
+```
+
+* ↩️  The OmniNode needs a runtime chainspec and in minimal case, we need to build the `minimal-runtime`,
+and then generate a chain spec based on it.
+
+
+```sh
+# Build the minimal runtime.
+cargo build -p minimal-template-runtime --release
+# Install chain-spec-builder if not installed already.
+cargo install staging-chain-spec-builder
+# Use chain-spec-builder to generate the chain_spec.json file based on the development preset.
+chain-spec-builder create --relay-chain "dev" --para-id 1000 --runtime \ 
+    <target/release/wbuild/path/to/minimal-template-runtime.wasm> named-preset development
+```
+
+* ⚙️  The `relay-chain` and `para-id` flags are extra bits of information to configure the node
+for the case of representing a parachain that is connected to a relay chain. For minimal they
+are not relevant, but they are mandatory flags nonetheless.
```

Comment:
I think we can skip them here, both flags are optional: 
https://github.com/paritytech/polkadot-sdk/blob/b4732add46910370443d092a3f479986060f6df5/substrate/bin/utils/chain-spec-builder/src/lib.rs#L169-L173

Reply:
(or maybe some other tool requires it - in that case would be good to clarify it)

Reply:
We'll still need them to be part of the chain spec to run minimal with polkadot-omni-node. It is just more convenient to add them with the command line instead of modifying the chain_spec.json file after generation.

Reply:
>  in that case would be good to clarify it)

Yep, I can clarify that not the flags are mandatory, but the fields in the chain_spec.json. Good catch!